### PR TITLE
Added Important Note #5295

### PR DESCRIPTION
**Pull request by j0rt3g4**
Added important note relative to issue #5295 

Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

### CODE REVIEW

File: windows/security/identity-protection/hello-for-business/hello-cert-trust-adfs.md [hunk 1]
```diff
@@ -499,6 +499,11 @@ Before you continue with the deployment, validate your deployment progress by re
 
 You need to verify the AD FS service has properly enrolled for an enrollment agent certificate template.  You can verify this is a variety ways, depending on if your service account is a normal user account or if the service account is a group managed service account.
 
+> [!IMPORTANT]
+> if after following the previous steps you are unable to validate that the devices are, in fact, being registrered automatically, there is a group Policy at:
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-cert-trust-adfs.md]: This sentence is cumbersome to read in its current phrasing and layout. I will make a suggestion, but JohanFreelancer9 might have a better suggestion.
```suggestion
> After following the previous steps, if you are unable to validate that the devices are, in fact, being registered automatically, there is a Group Policy at:
```

File: windows/security/identity-protection/hello-for-business/hello-cert-trust-adfs.md [hunk 2]
```diff
@@ -499,6 +499,11 @@ Before you continue with the deployment, validate your deployment progress by re
...
+> Computer Configuration -> Policies -> Administrative Templates -> Windows Components -> Device Registration -> "Register Domain Joined Computers As Devices", set it to enabled.
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-cert-trust-adfs.md]: From what I have observed previously on this page and earlier, the Microsoft Docs codestyle standard for this type of instruction is more like the following:
```suggestion
> **Computer Configuration > Policies > Administrative Templates > Windows Components > Device Registration >** "Register Domain Joined Computers As Devices". Set the policy to **Enabled**.
```
Worth noting: The dash character ( `-` ) used to create an ASCII arrow symbol will rarely stay aligned to point to the center of the `>` character in HTML text, when displayed on a web page using non-fixed fonts.

File: windows/security/identity-protection/hello-for-business/hello-cert-trust-adfs.md [hunk 3]
```diff
@@ -499,6 +499,11 @@ Before you continue with the deployment, validate your deployment progress by re
...
+> and the registration will happens automatically.
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-cert-trust-adfs.md]: grammar:
```suggestion
> and the registration will happen automatically.
```
(it is possible that the full sentence structure could be improved, but I don't have a better suggestion at the moment)

**Comment by illfated** [comment]
Oh well. I just thought of another possible improvement: Keep only the path description to the GPO (the continuous bold text) on its own line and move the start of the last sentence to the next line. (Not 100% sure if this is better, but I wanted to share the suggestion in case it is useful.)

**Comment by j0rt3g4** [comment]
@illfated  they are always useful

**Pull request merged**